exercises/nodejs/src/game.js

```diff
@@ -1,172 +1,198 @@
-import generator from 'random-seed'
+import generator from "random-seed";
 
-var Game = function () {
-  var players = new Array()
-  var places = new Array(6)
-  var purses = new Array(6)
-  var inPenaltyBox = new Array(6)
+let Game = function () {
+  const players = new Array();
+  const boardPositions = new Array(6);
+  const purses = new Array(6);
+  const inPenaltyBox = new Array(6);
 
-  var popQuestions = new Array()
-  var scienceQuestions = new Array()
-  var sportsQuestions = new Array()
-  var rockQuestions = new Array()
+  const popQuestions = new Array();
+  const scienceQuestions = new Array();
+  const sportsQuestions = new Array();
+  const rockQuestions = new Array();
 
-  var currentPlayer = 0
-  var isGettingOutOfPenaltyBox = false
+  const minimumPlayers = 2;
+  const numberOfCatagories = 4;
 
-  var didPlayerWin = function () {
-    return !(purses[currentPlayer] == 6)
-  }
+  let currentPlayer = 0;
+  let isGettingOutOfPenaltyBox = false;
 
-  var currentCategory = function () {
-    if (places[currentPlayer] == 0) { return 'Pop' }
-    if (places[currentPlayer] == 4) { return 'Pop' }
-    if (places[currentPlayer] == 8) { return 'Pop' }
-    if (places[currentPlayer] == 1) { return 'Science' }
-    if (places[currentPlayer] == 5) { return 'Science' }
-    if (places[currentPlayer] == 9) { return 'Science' }
-    if (places[currentPlayer] == 2) { return 'Sports' }
-    if (places[currentPlayer] == 6) { return 'Sports' }
-    if (places[currentPlayer] == 10) { return 'Sports' }
-    return 'Rock'
-  }
+  let didPlayerWin = function () {
+    return !(purses[currentPlayer] == 6);
+  };
 
-  this.createRockQuestion = function (index) {
-    return 'Rock Question ' + index
-  }
+  let currentCategory = function () {
+    if (boardPositions[currentPlayer] % numberOfCatagories === 0 ) {
+      return "Pop";
+    }
+    if (boardPositions[currentPlayer] % numberOfCatagories === 1) {
+      return "Science";
+    }
+    if (boardPositions[currentPlayer] % numberOfCatagories === 2) {
+      return "Sports";
+    }
+    return "Rock";
```

Comment:
Reducing the number of if statements is a very good refactor, but I would also combine the new 3 if statements into one "if-else if-else" statement that could be a bit easier to read